packages/cli/src/internal/options.ts

```diff
@@ -1193,7 +1193,7 @@ const parseInternal = (
       const extractKeyValue = (
         value: string
       ): Effect.Effect<[string, string], ValidationError.ValidationError> => {
-        const split = value.trim().split("=")
+        const split = value.trim().split(/=(.*)/, 2)
```

Comment:
I don't mind using Regex, but let's extract this to a constant in the global scope so the regex doesn't get re-created with each invocation.

Reply:
done